Description: document effect of http_proxy envvar on curl plugin
Author: Florian Schlichting <fsfs@debian.org> 
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=624774
Forwarded: http://mailman.blarg.de/pipermail/mpd-devel/2015-June/000363.html

--- a/doc/user.xml
+++ b/doc/user.xml
@@ -1777,6 +1777,13 @@
           Opens remote files or streams over HTTP.
         </para>
 
+        <para>
+          Note that unless overridden by the below settings (e.g. by
+          setting them to a blank value), general curl configuration
+          from environment variables such as http_proxy or specified
+          in ~/.curlrc will be in effect.
+        </para>
+
         <informaltable>
           <tgroup cols="2">
             <thead>
